src/include/ndpi_main.h

```diff
@@ -113,8 +113,15 @@ extern "C" {
   int ndpi_is_printable_buffer(u_int8_t const * const buf, size_t len);
   int ndpi_normalize_printable_string(char * const str, size_t len);
   int ndpi_is_valid_hostname(char * const str, size_t len);
-#define NDPI_ENTROPY_ENCRYPTED_OR_RANDOM(entropy) (entropy > 7.0f)
-  float ndpi_entropy(u_int8_t const * const buf, size_t len); 
+
```

Comment:
Could you add here a link to the presentation that you already linked from the commit msg, please?
It will be easy to find in the future....

Reply:
It's already done in `ndpi_utils.c` @ `ndpi_entropy2str()` .
Should I move it from there to the header file?